gfi/cli.py

```
@@ -261,50 +261,50 @@ def lucky(lang, min_stars, no_cache, use_graphql):
 
-            # Offer to generate card
-            try:
-                card_path = generate_card([(best_score, best_issue)], config["username"])
-                console.print(f"\n[dim]Card saved: {card_path}[/dim]")
-            except:
-                pass
+        # Offer to generate card
+        try:
+            card_path = generate_card([(best_score, best_issue)], config["username"])
+            console.print(f"\n[dim]Card saved: {card_path}[/dim]")
+        except:
```

Ruff (0.15.21)

[error] Do not use bare `except`

(E722)